internal/pkg/print/debug.go

```diff
@@ -141,13 +142,18 @@ func BuildDebugStrFromHTTPRequest(req *http.Request, includeHeaders []string) ([
 		return nil, fmt.Errorf("request is invalid")
 	}
 
-	status := fmt.Sprintf("request to %s: %s %s", req.URL, req.Method, req.Proto)
+	// unescape url in order to get rid of e.g. %40
+	unescapedURL, err := url.PathUnescape(req.URL.String())
+	if err != nil {
+		return nil, fmt.Errorf("request url could not be unescaped: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unescape request url: %w", err)
```
For better readability